scrapyd_k8s/__main__.py

```diff
@@ -13,4 +15,6 @@ def setup_logging():
 
 if __name__ == "__main__":
     setup_logging()
+    config = Config()
```

Comment:
Does this mean `Config` is instantiated twice, now?

Reply:
Yes, sorry, you are right, I can use the instance from api.py file